.github/workflows/main.yml

```diff
@@ -11,9 +11,9 @@ jobs:
         uses: ./ # Uses an action in the root directory
         with:
           styles: |
-            https://github.com/errata-ai/write-good/releases/latest/download/write-good.zip
-            https://github.com/errata-ai/Joblint/releases/latest/download/Joblint.zip
-          config: https://gist.githubusercontent.com/jdkato/171186055d382c6065273fe5a7eeeb61/raw/c6832996f5c77002142df70a3aa2be521fe747a7/.vale.ini
+            https://github.com/extenda/docker-vale/releases/download/latest/Extenda.zip
```

Comment:
Do we need both Extenda.zip and Google.zip?

Reply:
@davidberesfordER said we should use both in Extenda and we are using both in docker-vale repo. Docker-vale is only a developer tool apart from the style it provides for vale-action. The alternative would be to add the latest Google.zip URL in all workflows where we implement vale.

Reply:
I though we were bundling it with our zip...

Reply:
Only the Extenda style is released as a zip. The Google zip is added in the compose action in the actions repo.